src/screenshots/crop.js

```diff
@@ -64,7 +61,7 @@ function detectClippingArea (srcImage, { markSeed, clientAreaDimensions, cropDim
         const markIndex = srcImage.data.indexOf(mark);
 
         if (markIndex < 0)
-            throw new Error(renderTemplate(WARNING_MESSAGES.screenshotMarkNotFound, screenshotPath, markSeedToId(markSeed)));
+            return null;
```

Comment:
We should emit the warning if we can't find a mark on screenshots that contain browser UI. I guess we need to extract `crop by coordinates` code and make it usable by the `chome` provider without for searching for a mark.